UI/ClientUI.cpp

```diff
@@ -994,12 +994,12 @@ ClientUI* ClientUI::GetClientUI()
 { return s_the_UI; }
 
 void ClientUI::MessageBox(const std::string& message, bool play_alert_sound/* = false*/) {
-    GG::ThreeButtonDlg dlg(GG::X(320), GG::Y(200), message, GetFont(Pts()+2),
-                           WndColor(), WndOuterBorderColor(), CtrlColor(), TextColor(), 1,
-                           UserString("OK"));
+    auto dlg = /*TODO: Remove extra shared_ptr wrap after Wnd::Create converted to return shared_ptr*/std::shared_ptr<GG::ThreeButtonDlg>(GG::Wnd::Create<GG::ThreeButtonDlg>(GG::X(320), GG::Y(200), message, GetFont(Pts()+2),
+                                                   WndColor(), WndOuterBorderColor(), CtrlColor(), TextColor(), 1,
+                                                                                                                                                                              UserString("OK")));
```

Comment:
Indenting fixed.

It is reverted in the first few commits of https://github.com/freeorion/freeorion/pull/1632.